bucket/oripa.json

```
@@ -0,0 +1,36 @@
+{
+    "version": "0.36",
+    "homepage": "http://mitani.cs.tsukuba.ac.jp/oripa/",
+    "description": "Drawing software dedicated to designing the crease patterns of origami",
+    "license": "GPL-3.0-or-later",
+    "url": "https://github.com/oripa/oripa/releases/download/v0.36/oripa-0.36.jar#/oripa.jar",
+    "hash": "5e72f6aea5722e80a11fa8098cf3cd1f9c21be67014da39360abf640be194ebc",
+    "pre_install": [
+        "Expand-7zipArchive \"$dir\\oripa.jar\" -Switches 'icon\\oripa.gif'",
+        "Set-Content \"$dir\\oripa.bat\" 'start javaw -jar %~dp0\\oripa.jar %*' -Encoding ASCII",
```

Do not hardcore string, which are already variable, to just save few character of escaping.

```suggestion
        "Set-Content \"$dir\\oripa.bat\" \"start javaw -jar %~dp0\\$fname %*\" -Encoding ASCII",
```